testing/sdk_utils.py

```diff
@@ -165,6 +166,22 @@ def get_metadata() -> requests.Response:
                                    retry=False)
 
 
+def get_cluster_zones():
```

Comment:
We should probably add some retrying to this function. Example: https://github.com/mesosphere/dcos-commons/blob/4ef375d8c95272041d553c6549f7ed88c21889c1/testing/sdk_cmd.py#L276-L278

Reply:
Added retrying.

Reply:
Great!